lxd/storage_volumes.go

```diff
@@ -2691,7 +2691,7 @@ func addStoragePoolVolumeDetailsToRequestContext(s *state.State, r *http.Request
 	// Convert the volume type name to our internal integer representation.
 	volumeType, err := storagePools.VolumeTypeNameToDBType(volumeTypeName)
 	if err != nil {
-		return api.StatusErrorf(http.StatusBadRequest, err.Error())
+		return api.StatusErrorf(http.StatusBadRequest, "%s", err.Error())
```

Comment:
Is this part actually needed? Without the 3rd argument  the whole string is used right?

Reply:
It looks that starting from `golangci-lint` release `v1.60.1` (released last week) this now produces a `govet` error. See https://github.com/canonical/lxd/actions/runs/10416482319/job/28848800110.

It now also complains about cases where you do `fmt.Sprintf("static string")` which makes sense in my opinion as no format string is used here. See this microcluster PR as an example https://github.com/canonical/microcluster/pull/236.

Reply:
cool thanks!